airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -123,10 +132,13 @@ public class ConnectionManagerWorkflowImpl implements ConnectionManagerWorkflow
 
   private UUID connectionId;
 
+  private final MetricClient metricClient = MetricClientFactory.getMetricClient();
```

Comment:
this is where I'm not sure whether you'll have a metric client available. the metric client has to be initialized using MetricClient.initialize and the correct env variables need to be present where it's initialized.

Reply:
Ideally, that is something that will be handled by the framework.  I'll check on where we call initialize.  Either way, the code will just return the no-op client if it hasn't been initialized to avoid an NPE, so I think we are at least safe.

Reply:
@alovew I doubled checked and the `WorkerApp` calls initialize as the first thing it does.  Ultimately, this will be done by Micronaut but for now, it looks like everything is already set up for the Worker app.